server/src/test/java/io/spine/server/projection/ProjectionTransactionShould.java

```diff
@@ -140,6 +146,30 @@ protected void breakEntityValidation(
               .setShouldThrow(toThrow);
     }
 
+    @Ignore // The behavior is changed. See increment_version_on_event for the right behavior test
```

Comment:
It should be:
```
/**
 * This test is ignored as the expected behavior has been changed for the projection transactions.
 * 
 * <p>{@link #increment_version_on_event() Another test method} is created to test the new behavior. 
 * Please refer to it for more details.
 */
```

And please mention that this test hierarchy should be eliminated in the PM versioning issue you create(d).